README.md

```diff
@@ -287,6 +287,9 @@ If you don't want to use the setting `continuous_sync`, but still keep two insta
 ### Remove entries
 You can remove entries manually using the keybinds for `delete`. You can also delete the whole history with `:lua require('neoclip').clear_history()`.
 
+### Replace entries
```

Comment:
In terms of usability this seems quite low-level. Maybe it would be nice if there could be a keymap in the picker which opens the entry in a buffer which you can edit and when you save it it gets replaced. Alternatively maybe just some function that asks the user what entry to replace and what the new one should be using `vim.ui.input`? The former way seems nicer though as you can more easily actually "edit" an entry rather than replacing it.

Not sure if this goes beyond what you wanted to do but seems like just having the `replace` function might not be so usable?

Reply:
The former way is exactly the thing I was trying to implement when I first created this issue (#91). I didn't add it to this commit in fear of being too specific to my needs. I'll move my code over here. and you can take a look at how it's being done

Reply:
@AckslD It's done! I've implemented the edit function in [this](https://github.com/AckslD/nvim-neoclip.lua/pull/93/commits/ccd7e713163a60c818c2e4159a92d78b072007d0) commit.

I've mapped it to <c-e> in telescope, and called it "edit" (which to me just seems more intuitive than replace). It opens a new floating window with the entry's content and filetype set. Whenever the window is closed, the old entry's content is replaced, and telescope is reopened with the new entry pre-selected.

Take a look and let me know what you think!